src/input/mpegts/linuxdvb/linuxdvb_satconf.c

```diff
@@ -266,28 +266,28 @@ const idclass_t linuxdvb_satconf_class =
     {
       .type     = PT_U32,
       .id       = "min_rotor_move",
-      .name     = N_("Min Rotor Time (seconds)"),
+      .name     = N_("Min rotor time (seconds)"),
       .off      = offsetof(linuxdvb_satconf_t, ls_min_rotor_move),
       .opts     = PO_ADVANCED,
     },
     {
       .type     = PT_DBL,
       .id       = "site_lat",
-      .name     = N_("Site Latitude"),
+      .name     = N_("Site latitude"),
       .off      = offsetof(linuxdvb_satconf_t, ls_site_lat),
       .opts     = PO_ADVANCED,
     },
     {
       .type     = PT_DBL,
       .id       = "site_lon",
-      .name     = N_("Site Longitude"),
+      .name     = N_("Site longitude"),
       .off      = offsetof(linuxdvb_satconf_t, ls_site_lon),
       .opts     = PO_ADVANCED,
     },
     {
       .type     = PT_BOOL,
       .id       = "site_lat_south",
-      .name     = N_("Latitude Direction South"),
+      .name     = N_("Latitude direction south"),
```

Comment:
As in "this is below the equator", I presume? So is it "Southern hemisphere (latitude direction.."?